src/docs/release/breaking-changes/target-platform-linux-windows.md

```diff
@@ -0,0 +1,153 @@
+---
+title: Adding `linux` and `windows` to [`TargetPlatform`][] enum
+description: Two new values were added to the [`TargetPlatform`][] enum that could require additional cases in switch statements that switch on a [`TargetPlatform`][].
+---
+
+## Summary
+
+Two new values were added to the [`TargetPlatform`][] enum that could require additional cases in switch statements that switch on a [`TargetPlatform`][] and don't include a `default:` case.
+
+## Context
+
+Prior to this change, the [`TargetPlatform`][] enum only contained four values, and was defined like this:
+```dart
+enum TargetPlatform {
+  android,
+  fuchsia,
+  iOS,
+  macOS,
+}
+```
+
+A `switch` statement only needed to handle these cases, and desktop applications which wanted to run on Linux or Windows usually had a shim something like this in their main:
```

Comment:
Line is too long.

Also which => that

in their main => in their `main()` method:

Reply:
Done.